Tests/XcodeGenKitTests/SourceGeneratorTests.swift

```diff
@@ -559,13 +559,13 @@ class SourceGeneratorTests: XCTestCase {
                     - file.swift
                     - file.xcassets
                     - file.h
-                    - Info.plist
+                    - GoogleService-Info.plist
                     - file.xcconfig
                   B:
                     - file.swift
                     - file.xcassets
                     - file.h
-                    - Info.plist
+                    - Sample.plist
```

Comment:
I renamed these files in this test because the _real_ **Info.plist** is actually in group `C` defined on line 592 I assume because files in group `A` are all resources and files in group `B` had already been excluded automatically in the `TargetSource` definition. 

The previous logic ensured that **Info.plist** from group `C` had its `buildPhase` set to `.none` since it hadn't been previously overridden anywhere so it was falling back to the default value. I had to rename **A/Info.plist** and **B/Info.plist** because now the behaviour has changed so that only the resolved `INFOPLIST_FILE` gets it's build phase set to `.none` and since this test didn't define `info` in the spec (or the `INFOPLIST_FILE` value explicitly) then it falls back to the **first** file named **Info.plist** in the targets sources. 

Changing the names of the other two files just helps align with the _expected_ behaviour where the 'first' **Info.plist** is discovered as **C/Info.plist**.